### PR TITLE
123

### PR DESCRIPTION
**Pull request by ykolomiets**
Added new line with HelloWorld!!! Must have feature

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 Hello, world!!!
+Hello, world!!!
```
Comment (nazarnazar) [inline test.txt]: ?

Comment (ykolomiets) [reply test.txt]: Idi nahui) Vse ok

Comment (ykolomiets) [reply test.txt]: 123123123

Comment (ykolomiets) [reply test.txt]: 123123

Comment (ykolomiets) [reply test.txt]: @nazarnazar, suka zaebal

**Pull request merged**